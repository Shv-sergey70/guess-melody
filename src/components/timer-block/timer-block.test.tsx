import * as React from 'react';
import {configure, shallow} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import TimerBlock from "./timer-block";

configure({adapter: new Adapter()});

test(`Timer-block renders correctly`, () => {
  const tree = shallow(<TimerBlock/>);

  expect(tree).toMatchSnapshot();
});
