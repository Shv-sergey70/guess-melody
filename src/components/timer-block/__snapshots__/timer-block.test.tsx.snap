// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer-block renders correctly 1`] = `
<Fragment>
  <svg
    className="timer"
    viewBox="0 0 780 780"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      className="timer__line"
      cx="390"
      cy="390"
      r="370"
      style={
        Object {
          "filter": "url(#blur)",
          "transform": "rotate(-90deg) scaleY(-1)",
          "transformOrigin": "center",
        }
      }
    />
  </svg>
  <Connect(TimerContainer) />
</Fragment>
`;
